src/main/java/ui/components/KeyboardShortcuts.java

```diff
@@ -177,17 +164,31 @@ private static void addNonCustomizableShortcutKeys() {
         assignedKeys.add(MANAGE_ASSIGNEES);
     }
 
-    private static void getKeyboardShortcutsFromHashMap() {
-        markAsRead = getKeyCodeCombination("MARK_AS_READ");
-        markAsUnread = getKeyCodeCombination("MARK_AS_UNREAD");
-        scrollToTop = getKeyCodeCombination("SCROLL_TO_TOP");
-        scrollToBottom = getKeyCodeCombination("SCROLL_TO_BOTTOM");
-        scrollUp = getKeyCodeCombination("SCROLL_UP");
-        scrollDown = getKeyCodeCombination("SCROLL_DOWN");
-        leftPanel = getKeyCodeCombination("LEFT_PANEL");
-        rightPanel = getKeyCodeCombination("RIGHT_PANEL");
-        upIssue = getKeyCodeCombination("UP_ISSUE");
-        downIssue = getKeyCodeCombination("DOWN_ISSUE");
+    public static KeyCodeCombination getKeyboardShortcut(KeyboardShortcutKey key) {
+        switch (key) {
+            case MARK_AS_READ:
+                return getKeyCodeCombination("Mark_AS_READ");
```

Comment:
This repeats ourselves a bit. :stuck_out_tongue: Could we merge this method and `getKeyCodeCombination`?